x-pack/plugins/security_solution/common/api/entity_analytics/entity_store/entities/list_entities.gen.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/*
+ * NOTICE: Do not edit this file manually.
+ * This file is automatically generated by the OpenAPI Generator, @kbn/openapi-generator.
+ *
+ * info:
+ *   title: Entities List Schema
+ *   version: 2023-10-31
+ */
+
+import { z } from '@kbn/zod';
+import { ArrayFromString } from '@kbn/zod-helpers';
+
+import { EntityType, InspectQuery } from '../common.gen';
+import { UserEntityRecord, HostEntityRecord } from './common.gen';
+
+export type ListEntitiesRequestQuery = z.infer<typeof ListEntitiesRequestQuery>;
+export const ListEntitiesRequestQuery = z.object({
+  sort_field: z.string().optional(),
+  sort_order: z.enum(['asc', 'desc']).optional(),
+  page: z.coerce.number().int().min(1).optional(),
+  per_page: z.coerce.number().int().min(1).max(10000).optional(),
+  /**
+   * An ES query to filter by.
+   */
+  filterQuery: z.string().optional(),
+  entities_types: ArrayFromString(EntityType),
+});
+export type ListEntitiesRequestQueryInput = z.input<typeof ListEntitiesRequestQuery>;
+
+export type ListEntitiesResponse = z.infer<typeof ListEntitiesResponse>;
+export const ListEntitiesResponse = z.object({
+  records: z.array(z.union([UserEntityRecord, HostEntityRecord])),
```

Comment:
Would `entities` be nicer? I think records is an asset criticality thing unless we use it in all our APIs?

Reply:
I don't think it is widely used inside security solutions, but we also use this terminology for `EntityRiskScoreRecord`.

But I don't mind changing it. What do you think would be better? `UserEntity` and `HostEntity`? I would like to avoid just `User` and `Host` to disambiguate.

Reply:
Yeah I think remove record, it doesnt really add anything, saves us (GH copilot ;) ) a few keyboard strokes

Reply:
➕